### PR TITLE
[FTAMS102020] Irene Asensio

### PR DESCRIPTION
**Pull request by ihatepineapple**
On route /drones, the cards from bootstrap are not displaying in a row. 
This is a styling issue but I had it before, any comments on how to solve it?

### CODE REVIEW

File: views/drones/list.hbs [hunk 1]
```diff
@@ -1 +1,27 @@
...
+                </form>
+            </div>
+        </div>
+    </div>
+         </div>
+     </div>
+</div>
+
+{{/each}} 
+<a href="/drones/create" class="btn btn-warning">Add Drones</a>
```
Comment (wax-bear) [inline views/drones/list.hbs:27]: I would name this button "Add drone" (singular instead of plural).

File: views/layout.hbs [hunk 2]
```diff
@@ -6,14 +6,33 @@
...
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <link rel="stylesheet" href="/stylesheets/style.css" />
   </head>
 
   <body>
 
+  
+
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="#"><b>{{title}}</b></a>
```
Comment (wax-bear) [inline views/layout.hbs:18]: I would href this to the home or the drone route.

**Review by wax-bear — COMMENTED** [review]
Great stuff Irene!
I like the styling of the pages.
I only placed two minor comments regarding naming and href.

File: views/drones/list.hbs (hunk 1 above)
Comment (ihatepineapple) [reply views/drones/list.hbs:27]: noted

File: views/layout.hbs (hunk 2 above)
Comment (ihatepineapple) [reply views/layout.hbs:18]: noted as well. Any comments on how to solve the flex issue on the drones list page?

Comment (wax-bear) [reply views/layout.hbs:18]: Could you enlighten me on the flex issue? :)

**Pull request closed without merging**